### ISSUE TITLE
training with custom loss function, or assigning weights to classes

### ISSUE THREAD

**Issue by fhamborg** [explosion/spaCy#6873]
Hi! 

Is it possible to define a custom loss function when training spacy's transformers? Specifically, I'm asking because the classes in my dataset are imbalanced, and I'd like to weight each class differently. In pytorch, you could define a weight for each class in various loss functions, such as `nn.CrossEntropyLoss(weight=...)` 

How would that work in spacy-transformers?

Thanks in advance for your answer! :-)

Cheers,
Felix
